### DISCUSSION TITLE
機器狗會游泳，美國軍用機械犬 Vision 60 推出新功能

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#672] [科技新知]
有了軍用機器犬,可以節省訓練狗狗的時間,也可以減少發生意外,導致軍犬失去生命.

新聞來源 : https://technews.tw/2022/06/14/ghost-robotics-vision60-got-amphibious-function/

**Comment by Hchejxnc** [comment]
機械犬的發明感覺在救援上也能發揮很大的功能

**Comment by harrylee1971** [comment]
但取代掉軍犬，反而原本可以成為軍犬的狗，他們有可能會被原本的收容所給安樂死

**Comment by ghost** [comment]
我覺得未來的戰爭應該會去人類化，也就是全部改用機器，因為人際怕死，又想用讓人死的方式來解決問題，所以機器化是一舉兩得的作法。我認為軍用機器犬或許就是去人類戰爭的第一步，不過還是希望這個世界沒有戰爭
